### PR TITLE
feat(AWS MSK): Add support for SASL/SCRAM authentication

### PR DESCRIPTION
**Pull request by Arielgordon123**
Add missing SourceAccessConfiguration in the event source mapping config

Closes: #9224

### CODE REVIEW

**Comment by pgrzesik** [comment]
Hello @Arielgordon123 - thanks a lot for the proposed change. Is there a reason why you changed the proposed configuration scheme in comparison to what was agreed upon in the related issue?

**Comment by Arielgordon123** [comment]
Hi @pgrzesik I've fixed that,
now it would work like the proposal in the related issue

File: test/integration/aws/infra-dependent/msk.test.js [hunk 1]
```diff
@@ -58,6 +58,12 @@ describe('AWS - MSK Integration Test', function () {
                   topic: topicName,
                   batchSize: 100,
                   maximumBatchingWindow: 3,
+                  SourceAccessConfigurations: [
```
Comment (pgrzesik) [inline test/integration/aws/infra-dependent/msk.test.js:61]: Let's remove this setting here

File: scripts/test/integration-setup/cloudformation.yml [hunk 2]
```diff
@@ -202,11 +202,41 @@ Resources:
         - ConsoleAccess: 'true'
           Password: !Ref RabbitMQPassword
           Username: !Ref RabbitMQUser
-
+  MSKKey:
```
[the remark this answers is not in the document]
Comment (Arielgordon123) [reply scripts/test/integration-setup/cloudformation.yml]: @pgrzesik removed

File: test/integration/aws/infra-dependent/msk.test.js (hunk 1 above)
Comment (Arielgordon123) [reply test/integration/aws/infra-dependent/msk.test.js]: removed

**Review by pgrzesik — APPROVED** [review]
Thank you @Arielgordon123, it looks great 👍

**Comment by pgrzesik** [comment]
Hey @valdirluiz - no specific date yet, but probably later this week or early next week.

**Pull request merged**